GiniVision/Classes/Core/PageStateView.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  PageStateView.swift
+//  GiniVision
+//
+//  Created by Enrique del Pozo Gómez on 4/13/18.
+//
+
+import Foundation
+
+final class PageStateView: UIView {
+    
+    enum State {
+        case success, failure, loading
```

Comment:
I'm just being picky, but the naming is not consistent. Success and failure are nouns and then you have "loading", which is not. And from a spoken language perspective, the question is "What's the state of this page?" an answer would be "It failed / It's loading / it succeeded". I think the general convention would be to have:

- completed
- failed
- loading

Reply:
You are right, I was biased with the naming and didn't pay attention to it. 👍